SampleApps/CarthageTest/CarthageTest.xcodeproj/project.pbxproj

```diff
@@ -413,7 +373,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.braintreepayments.CarthageTest;
+				PRODUCT_BUNDLE_IDENTIFIER = com.braintreepayments.CarthageTestApp;
```

Comment:
I think all of these CarthageApp changes aren't next-major-version specific, and we'd also want them on master/v5.

At this point it might just be easier to duplicate these changes in a separate PR, since I know we want to get the beta out.